app/assets/stylesheets/content/_tables.sass

```diff
@@ -60,11 +60,21 @@ table
 
   &.workflow-table
     margin-bottom: 16px
+    border-collapse: separate
```

Comment:
Why do you choose separate border?
All border I can see still look collapsed.

I have to admit that in IE11 the bottom border disappears when disabling this setting (as well as some other small pixel movements), but I suspect this has other reasons...